### ISSUE TITLE
Calculate and track estimated calorie expenditure

### ISSUE THREAD

**Issue by Wall-Dough** [ScienceSquad/Health#20]
_< no comment >_

---
- **Estimated Time:** _8 hours._
- **Prerequisites:** _Sensor Integration, Nutrition Module, Calendar View, Machine Learning, Calorie Database._

**Comment by colinrford** [comment]
Should we expand this beyond just walking/running? I.e., if #8 also kept track of calorie expenditure over the course of various exercises.

**Comment by Wall-Dough** [comment]
Yes, I think we should be sure that we aren't segregating the different modules. They should all be related to each other, for sure.

It might be a good design to make something like calorie tracking a separate module that still references data entered in the other modules.

Know calories burned from running/cycling/exercising/walking as well as calories taken in from diet.

Unfortunately I don't know if we could be so savvy as to add base metabolism as a variable. That would be really nice and far more accurate.

**Comment by avaidyam** [comment]
Some of @brandonschabell's issues have this idea too. I know Google Fit has API to track running/walking/biking/walking up steps and their transitions. The calories burned will be a function of time, height/weight, and exercise mode, I think.

**Comment by Brahmancer** [comment]
#22 could be useful.

**Comment by colinrford** [comment]
@Wall-Dough @avaidyam @Brahmancer @brandonschabell @MrJohnson no one has worked on this have they

**Comment by Brahmancer** [comment]
Looking at the Sprint Planning Document, that was a part of the Sprint Backlog so we can push this back to a later sprint.

**Comment by avaidyam** [comment]
@Brahmancer can you post to me on Slack the stuff on my backlog for this sprint? It seems like it's not synced.

**Comment by Brahmancer** [comment]
@avaidyam Sent to you your user stories via direct message on slack.

**Comment by colinrford** [comment]
I guess this is being pushed back to **Sprint 3**?

**Comment by Brahmancer** [comment]
It doesn't have to be. If someone wants to take it for Sprint 3, we can.

**Comment by avaidyam** [comment]
Sprint 3 it is!
